### PR TITLE
adding updates to 01-tidiness lesson

### PR DESCRIPTION
**Pull request by tracykteal**
This is a work in progress. Wait to merge.

### CODE REVIEW

**Comment by tracykteal** [comment]
This is now ready for review.

File: _episodes/01-tidiness.md [hunk 1]
```diff
@@ -3,79 +3,101 @@ title: "Data Tidiness"
...
+>
+{: .callout}
 
-1. What are some errors you can spot in the data?
--  Typos, missing data, inconstancies?
-2. What improvements could be made to the choices in naming?
-3. What are some errors in the spreadsheet that would be difficult to spot? Is there anyway you can test this?
+### Structuring data in spreadsheets
 
+Independent of the what type of data you're collecting there are standard ways to enter that data into the spreadsheet, to make it easier to analyze later. We often enter data that makes it easy for us as humans to read and work with it, because we're human! Computers need data structured in a way that they can use it, so to use this data in a computational workflow, we need to think like computers when we use spreadsheets.
```
Comment (ErinBecker) [inline _episodes/01-tidiness.md]: the what --> extra word

File: _episodes/01-tidiness.md [hunk 2]
```diff
@@ -3,79 +3,101 @@ title: "Data Tidiness"
...
 
-## **B. Retrieving samples from the facility**
+The cardinal rules of using spreadsheet programs for data:
 
-When the data come back from the facility, you will receive some documentation (metadata) as well as the sequence files themselves. Download and examine the following file - here provided both as an excel and a text file:
+- Put all your variables in columns - the thing you’re measuring, like ‘weight’ or ‘temperature’.
```
Comment (ErinBecker) [inline _episodes/01-tidiness.md]: change 'weight' and 'temp' to variables that would be included in genomic metadata table. E.g. 'strain name' and 'library prep method'

File: _episodes/01-tidiness.md [hunk 3]
```diff
@@ -3,79 +3,101 @@ title: "Data Tidiness"
...
 
-Before analysis of data has begun, there are already many potential areas for errors and omissions. Keeping organized and keeping a critical eye can help catch mistakes. 
+Fear not! If you have already started your project, and it's not set up this way, there are still opportunities to make updates. One of the biggest challenges is tabular data that isn't formatted so computers can use it, or has inconsistencies that make it hard to analyze.
 
-One of Data Carpentry's goals is to help you achieve *competency* in working with bioinformatics. This means that you can accomplish routine tasks, under normal conditions, in an acceptable amount of time. While an expert might be able to get to a solution on instinct alone - taking your time, using Google, and asking for help are all valid ways of solving your problems. As you complete the lessons you'll be able to use all of those methods more efficiently.
+There are guidelines on how to structure your data, outlined in our [Data Carpentry Ecology spreadsheet lesson](http://www.datacarpentry.org/OpenRefine-ecology-lesson/)
```
Comment (ErinBecker) [inline _episodes/01-tidiness.md]: I think these guidelines are now included earlier in this episode, so maybe we don't need to reference the Ecology lessons here.

File: _episodes/01-tidiness.md [hunk 4]
```diff
@@ -3,79 +3,101 @@ title: "Data Tidiness"
...
+- "How to collect and structure the data about your sequencing data"
 objectives:
 - "Think about and understand the types of metadata a sequencing experiment will generate."
-- "Make decisions about how (if) data will be stored, archived, shared, etc."
-- "Anticipate strategies we will need to learn in the rest of the lesson set."
+- "Understand the importance of metadata and potential metadata standards"
+- "Explore common formatting challenges in spreadsheet data"
 keypoints:
-- "First key point."
+- "Metadata is key for you and others to be able to work your data"
```
Comment (ErinBecker) [inline _episodes/01-tidiness.md]: work --> work with

**Comment by ErinBecker** [comment]
@tracykteal - I meant to put in a PR to your branch with a few changes, but I think I ended up push directly to your branch. Could you please review my changes and go ahead and merge if they look ok to you? 

I fixed a few missing/extra words and changed some of the examples to be more directly applicable to genomics data.

**Comment by tracykteal** [comment]
Looks like the did get merged already. I'll put in my PR to fix your comment, then address those ones in a different PR.

**Comment by tracykteal** [comment]
Updated the language around the spreadsheet ecology lessons. Looks like your changes already went into the branch. So, I'll put in a PR to this PR on those.

**Comment by tracykteal** [comment]
I think all the suggested changes have been incorporated now.

**Comment by ErinBecker** [comment]
Thanks @tracykteal - LGTM.

**Pull request merged**